### ISSUE TITLE
Failed to run revapi-maven-plugin version 0.9.3

### ISSUE THREAD

**Issue by monperrus** [revapi/revapi#100]
We have just updated Revapi but we have an error. Any idea to fix it?

```
[ERROR] Failed to execute goal org.revapi:revapi-maven-plugin:0.9.3:report (default-cli) on project 
spoon-core: Execution default-cli of goal org.revapi:revapi-maven-plugin:0.9.3:report failed: An API 
incompatibility was encountered while executing org.revapi:revapi-maven-plugin:0.9.3:report: 
java.lang.NoSuchMethodError: java.nio.CharBuffer.flip()Ljava/nio/CharBuffer;
```

```
# mvn -version
Apache Maven 3.5.2 (138edd61fd100ec658bfa2d307c43b76940a5d7d; 2017-10-18T09:58:13+02:00)
Maven home: /opt/apache-maven-3.5.2
Java version: 1.8.0_151, vendor: Oracle Corporation
Java home: /opt/jdk1.8.0_151/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "3.16.0-4-amd64", arch: "amd64", family: "unix"
```

**Comment by metlos** [comment]
Yep, this is related to #99. I'm using Java 9 for the build and use animal sniffer to check that I'm not using any Java 9 specific APIs. Apparently, this doesn't work too well.

But the `-release` flag should solve this issue. A new release is coming out hopefully today..

**Comment by monperrus** [comment]
> A new release is coming out hopefully today.. 
Excellent. Looking forward to it.

**Comment by metlos** [comment]
This should be fixed with `revapi-maven-plugin-0.9.4`.

**Comment by monperrus** [comment]
Thanks, it works. But we now have https://github.com/revapi/revapi/issues/101
